src/controller/CHIPDeviceController.h

```diff
@@ -91,6 +91,12 @@ struct ControllerInitParams
     DeviceAddressUpdateDelegate * mDeviceAddressUpdateDelegate = nullptr;
 #endif
     OperationalCredentialsDelegate * operationalCredentialsDelegate = nullptr;
+
+    Crypto::P256Keypair * ephemeralKeypair = nullptr;
```

Comment:
Document what this keypair is supposed to be for, please.  Right now, as an API consumer, I have no idea what I would set it to.  In particular, if it needs to be related to the certs in some way, explain how, please.